app/internal/page/certificateprovider/who_is_eligible.go

```diff
@@ -0,0 +1,32 @@
+package certificateprovider
+
+import (
+	"net/http"
+
+	"github.com/ministryofjustice/opg-go-common/template"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/page"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/sesh"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/validation"
+)
+
+type whoIsEligibleData struct {
+	App    page.AppData
+	Lpa    *page.Lpa
+	Errors validation.List
+}
+
+func WhoIsEligible(tmpl template.Template, lpaStore LpaStore, store sesh.Store) page.Handler {
```

Comment:
Annoyingly needed this handler as `Guidance()` requires a valid session to work.